app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/componentHealthCheckControls/componentHealthCheckControls.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { FieldArray } from 'redux-form';
+import { validate } from 'common/utils';
+import { COMPONENT_HEALTH_CHECK } from 'common/constants/widgetTypes';
+import { FieldProvider } from 'components/fields/fieldProvider';
+import { FiltersControl, InputControl } from '../controls';
+import { ITEMS_INPUT_WIDTH } from '../constants';
+import { AttributesFieldArrayControl } from '../controls/attributesFieldArrayControl/attributesFieldArrayControl';
```

Comment:
You can export it in `controls` and import in [this](https://github.com/reportportal/service-ui/pull/1988/files#diff-ce11d5becdc80e4866f614b94962a597R9) line